examples/v2-minimal/packages/contracts/types/ethers-contracts/IWorld.ts

```diff
@@ -56,6 +56,8 @@ export interface IWorldInterface extends utils.Interface {
     "registerTable(bytes16,bytes16,bytes32,bytes32)": FunctionFragment;
     "registerTableHook(bytes16,bytes16,address)": FunctionFragment;
     "revokeAccess(bytes16,bytes16,address)": FunctionFragment;
+    "setEphemeralRecord(bytes32,bytes32[],bytes)": FunctionFragment;
+    "setEphemeralRecord(bytes16,bytes16,bytes32[],bytes)": FunctionFragment;
```

Comment:
@alvrs @dk1a I think the answer is no, but does our module system support opt-in features like this but without the need to go through the hooks path? Just worried about extending the base world with ephemeral events if we're running low on contract size and have other array methods planned.

Reply:
yeah if you mean the as-yet-unimplemented plugin hooks, modules don't need them for this. `core` module is a good example, as all it does is add more functions to `World` because of contract size issues